pkg/settings/setting.go

```diff
@@ -538,6 +539,14 @@ func IsReleaseServerVersion(serverVersion string) bool {
 	return true
 }
 
+// IsPlainIPV6 will return true if the given address is a plain IPV6 address and not encapsulated or similar.
+func IsPlainIPV6(address string) bool {
+	if net.ParseIP(address) != nil && strings.Count(address, ":") >= 2 {
```

Comment:
```suggestion
	if ip := net.ParseIP(address); ip != nil && len(ip) == net.IPv6len {
```
May actually be more suitable, but I understand if we don't want to change. FWIW that's how the net package does it: https://cs.opensource.google/go/go/+/refs/tags/go1.23.0:src/net/ip.go;l=229